### PR TITLE
Only show first line of multilines post when showing "commented on..."

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Only show first line of multilines post when showing "commented on..."

#### Ticket Link
Jira ticket: [MM-11927](https://mattermost.atlassian.net/browse/MM-11927)

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Added or updated unit tests (required for all new features)

### CODE REVIEW

**Comment by jespino** [comment]
I think we are changing the meaning of "RemoveMarkdown" and "StripMarkdown" here. and they ar used in other places, maybe with different expectations. Maybe we can create a new function.

**Comment by saturninoabril** [comment]
@jespino Right.  I renamed `stripMarkdown` to `generateShortMessage` which under the hood is removing markdown and returning the first line of the message.  `stripMarkdown` or `generateShortMessage` is being used for notification and "commented on" which basically expect the same result.

**Comment by jespino** [comment]
There is other usage of `RemoveMarkdown` in formatText when you pass the option of `options.removeMarkdown`, for now I don't see any place using that option, but still I think we need to do something with the `RemoveMarkdown` name to be less error prone in the future.

File: utils/markdown/remove_markdown.js [hunk 1]
```diff
@@ -5,31 +5,31 @@ import marked from 'marked';
 
 export default class RemoveMarkdown extends marked.Renderer {
     code(text) {
-        return ' ' + text.split('\n').join(' ');
+        return text.split('\n')[0];
```
Comment (hmhealey) [inline utils/markdown/remove_markdown.js]: I don't think all these splits in the renderer should be necessary when you're doing it in `stripMarkdown` after this has all been done

**Pull request closed without merging**